tap_dynamics/__init__.py

```diff
@@ -89,21 +89,16 @@ def __call__(self, r):
 @singer.utils.handle_top_exception(LOGGER)
 def main():
     parsed_args = singer.utils.parse_args(REQUIRED_CONFIG_KEYS)
-
-    try:
+   
+    if parsed_args.config.get('full_url'):
```

Comment:
please change it so it also checks the content of the field, probably the frontend will return an empty string when the field is not filled. You can add a second if to avoid this issue.
```
if parsed_args.config.get('full_url'):
    if parsed_args.config['full_url']:
        url = parsed_args.config['full_url']
```

Reply:
Okay will add!